apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -22,6 +23,33 @@ test.afterEach(async ({ users }) => {
   await users.deleteAll();
 });
 
+test("check SSR and OG", async ({ page, users }) => {
+  const name = "Test User";
+  const user = await users.create({
+    name,
+  });
+  const [response] = await Promise.all([
+    // This promise resolves to the main resource response
+    page.waitForResponse(
+      (response) => response.url().includes(`/${user.username}/30-min`) && response.status() === 200
+    ),
+
+    // Trigger the page navigation
+    page.goto(`/${user.username}/30-min`),
+  ]);
+  const ssrResponse = await response.text();
```

Comment:
Important to verify on raw text as on client it get's fixed as soon as react hydration kicks in.